_posts/blog/2017-07-18-dwh-part2-we-need-a-storage.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution<br/> Part 2: We Need a Storage"
+description: 
+modified:
+categories: blog
+author: robert_bordo
+excerpt: We would like to share our second part towards a custom data warehouse solution. This time it's all about selecting a proper storage.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake, Storage, Redshift, S3, Snowflake]
+image:
+date: 2017-07-18T00:39:55-04:00
+---
+
+> In the beginning we created a cluster. And the cluster was without form, and void; and nulls were upon the face of the storage.
```

Comment:
is this a quote? if yes from whom? 
And the cluster was without form, and void <- get rid of the and in the beginning and the second sentence misses a verb